### ISSUE TITLE
[Bug Hunt] Screen Reader is not working for Images (Images Accessibility feature) 

### ISSUE THREAD

**Issue by greenmug-md** [topcoder-platform/u-bahn-app#527]
Video : https://drive.google.com/file/d/1q0Q_Nuu6ZydidrhQ0T5A3JuZArM8BvMG/view?usp=sharing

Reproduction Steps:
	1.	Open the Application https://skill-search.topcoder-dev.com/
	2.	Click on any image to understand the text
	3.	Screen Reader should be able to work with the images as well such as Tabs Icons , Search Icon etc
	
Expected Result:  Screen Reader should work for images 

Device/OS: Laptop | MACOS  (64bit)
Browser Information: Chrome 83.0.4103.116 | Firefox 78.0.2 | IE11

**Comment by callmekatootie** [comment]
Checked with PM - This is not needed at the moment.
